components/wishlist/WishlistButton/WishlistButton.tsx

```diff
@@ -31,10 +31,8 @@ const WishlistButton: FC<Props> = ({
   const itemInWishlist = data?.items?.find(
     // @ts-ignore Wishlist is not always enabled
     (item) =>
-      item.product_id === Number(productId) &&
-      (item.variant_id as any) === Number(variant.id)
```

Comment:
This change breaks all other providers types

Reply:
Hi @goncy! Sorry, I don't understand that code. Our `productId` is a string, and  I can only get NaN from the Number function, and I'm not able to activate the button. How could I avoid that?

Reply:
In that case can you create a PR for this change separately so we can leave this PR separated from the core? And I can deploy it with other providers to see if it works

Reply:
Makes sense, I will do it

Reply:
`Number(productId)` will return NaN 

You could remove the `Number(productId)` and change it to `productId`